apis/kubedb/v1alpha2/constants.go

```diff
@@ -507,6 +507,84 @@ const (
 	InternalUsersCredentialsSyncedSuccessfully = "InternalUsersCredentialsSyncedSuccessfully"
 )
 
+const (
+	KafkaRESTPortName             = "http"
+	KafkaControllerPortName       = "controller"
+	KafkaInternalPortName         = "internal"
+	KafkaBrokerClientPortName     = "broker-client"
```

Comment:
`KafkaPortName<something>`.  This will allow to search on the ide more convenient

Reply:
Updated